api/edge_api/identities/views.py

```diff
@@ -168,9 +168,7 @@ def update_traits(self, request, *args, **kwargs):
         identity.update_traits([trait])
         Identity.dynamo_wrapper.put_item(build_identity_dict(identity))
         data = trait_schema.dump(trait)
-        send_identity_update_message.delay(
-            args=(environment.api_key, identity.identifier)
-        )
+        send_identity_update_message(environment, identity.identifier)
```

Comment:
This looks like it's doing something synchronously now? I'm sure that the async logic is just handed elsewhere now and I just haven't got that far yet but could we name this function better?

Reply:
I think that how send_identity_message actually send the message (by creating a task for example) is just an implementation detail of the service?

Reply:
Yeah, I tend to agree on second thoughts.